frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -153,13 +153,23 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
               </DashboardDescriptionListGroup>
             </>
           )}
+          <Title style={{ marginTop: '1em' }} headingLevel={TextVariants.h3}>
+            Source model format
+          </Title>
```

Comment:
@manaswinidas this is a real nitpick, but can you follow the same pattern above with the "Model location" title where we close the `</DescriptionList>` before the `Title` and then open a new `<DescriptionList isFillColumns>`? And then can we add some additional margin between the two source model format fields and the Author field, since it's not part of Source model format? Maybe putting those last few fields in their own `DescriptionList` would also do that. Right now you can't tell where the "Source model format" section ends.

Reply:
Added `DescriptionList` with margin.

Introducing a`DescriptionList` between version and author without margin:
<img width="527" alt="Screenshot 2024-10-11 at 1 29 04 PM" src="https://github.com/user-attachments/assets/43093f60-dd89-49bd-8813-6ebdfb2f097e">

Introducing a`DescriptionList` between version and author with `2em` margin(current):
<img width="469" alt="Screenshot 2024-10-11 at 1 37 11 PM" src="https://github.com/user-attachments/assets/2d1288e8-9c58-425c-b5a5-0dbd881bb198">

Without`DescriptionList` between version and author:
<img width="322" alt="Screenshot 2024-10-11 at 1 25 29 PM" src="https://github.com/user-attachments/assets/1ea8f21c-8369-4f94-8145-414cc2446d99">